README.md

```diff
@@ -1,10 +1,83 @@
-### Running locally
+# nextjs_dev_folio
 
-```
-npm run dev
-```
+## About
+
+`nextjs_dev_folio` is a personal portfolio website built using Next.js. This project allows developers to showcase their skills, projects, and personal information in an organized and visually appealing manner. This repository is open for **Hactoberfest 2023!!**
+
+## Getting Started
+
+### Running Locally
+
+To run the project locally, follow these steps:
+
+1. Clone this repository to your local machine:
+
+   ```bash
+   git clone https://github.com/danger-ahead/nextjs_dev_folio.git
+   ```
+
+2. Navigate to the project directory:
+
+   ```bash
+   cd nextjs_dev_folio
+   ```
+
+3. Install the project dependencies:
+
+   ```bash
+   npm install
```

Comment:
we're using `pnpm`. Can you please replace `npm` with `pnpm`?

Reply:
Okay i will make the changes